elasticsearch/common/src/main/java/org/apache/metamodel/elasticsearch/common/ElasticSearchUtils.java

```diff
@@ -78,15 +86,15 @@ public static QueryBuilder getExistsQuery(String fieldName) {
         return getFilteredQuery("exists", fieldName);
     }
 
-    public static List<Object> getSourceProperties(final MutableTable table) {
+    public static Object getMappingSource(final MutableTable table) {
```

Comment:
Hmm I chose for Object because I felt that the data type was irrelevant - or even an implementation detail. The 'source' object is some kind of magical object anyway without much type safety. But on the other hand, it seems that the `source(Map)` method is different in behaviour from `source(Object)` so that's a good case for changing this method to return map.